src/components/course-calendar/course-month.cjsx

```diff
@@ -0,0 +1,50 @@
+moment = require 'moment'
+React = require 'react'
+BS = require 'react-bootstrap'
+
+{Calendar, Month, Week, Day} = require 'react-calendar'
+
+CourseMonth = React.createClass
+  displayName: 'CourseMonth'
+
+  getInitialState: ->
+    date: moment()
+
+  handleNextMonth: (clickEvent) ->
+    clickEvent.preventDefault()
+    @setState(
+      date: @state.date.clone().add(1, 'month')
+    )
+
+  handlePreviousMonth: (clickEvent) ->
+    clickEvent.preventDefault()
+    @setState(
+      date: @state.date.clone().subtract(1, 'month')
+    )
+
+  render: ->
+
+    # TODO see about doing a PR to the library after their react update to enable
+    # modifying MonthHeader as a feature
+    MonthHeader =
+      <BS.Row className='month-header'>
+        <BS.Col xs={4}>
+          <a href="#" className='month-header-control previous' onClick={@handlePreviousMonth}><</a>
+        </BS.Col>
+        <BS.Col xs={4} className='month-header-label'>{@state.date.format('MMMM YYYY')}</BS.Col>
+        <BS.Col xs={4}>
+          <a href="#" className='month-header-control next' onClick={@handleNextMonth}>></a>
+        </BS.Col>
+      </BS.Row>
+
+
+    <BS.Grid>
+      {MonthHeader}
+      <BS.Row>
+        <BS.Col xs={12}>
+          <Month date={@state.date}/>
```

Comment:
Ok, the following works (but changes the calendar styling a bit):

```coffee
    hw4 = <span onClick={-> alert('Clicked HW 4')}>HW4</span>

    <BS.Grid>
      {MonthHeader}
      <BS.Row>
        <BS.Col xs={12}>
          <Month date={@state.date}>
            <Day date={moment('2015-04-11')} dayAgenda={true} >{hw4}</Day>
          </Month>
        </BS.Col>
      </BS.Row>
    </BS.Grid>
```

![image](https://cloud.githubusercontent.com/assets/253202/7102839/6c4b225c-e05d-11e4-8f64-081c36df22d9.png)